### PR TITLE
check if property is empty in request

### PR DESCRIPTION
**Pull request by pxpm**
### BEFORE - What was wrong? What was happening before this PR?

We were only covering the case where empty strings would be converted to `null` by using Laravel `ConvertEmptyStringsToNull` middleware. 

As reported in https://github.com/Laravel-Backpack/CRUD/issues/4727 , disabling that middleware had side effects on our pagination because the null coalescing operator does not understand empty the same as null.

### AFTER - What is happening after this PR?

we use `empty()` so that we cover empty strings or null values.

### How can we test the before & after?

Enable/disable the middleware and test the ListOperation

### CODE REVIEW

File: src/app/Http/Controllers/Operations/ListOperation.php [hunk 1]
```diff
@@ -97,7 +97,7 @@ public function search()
 
         // if show entry count is disabled we use the "simplePagination" technique to move between pages.
         if ($this->crud->getOperationSetting('showEntryCount')) {
-            $totalEntryCount = (int) (request()->get('totalEntryCount') ?? $this->crud->getTotalQueryCount());
+            $totalEntryCount = (int) (empty(request()->get('totalEntryCount')) ? $this->crud->getTotalQueryCount() : request()->get('totalEntryCount'));
```
[the remark this answers is not in the document]
Comment (basselhossam) [reply src/app/Http/Controllers/Operations/ListOperation.php:100]: @pxpm I meant `request()->get('totalEntryCount') ?: $this->crud->getTotalQueryCount()`

[the remark this answers is not in the document]
Comment (basselhossam) [reply src/app/Http/Controllers/Operations/ListOperation.php:100]: image

[the remark this answers is not in the document]
Comment (pxpm) [reply src/app/Http/Controllers/Operations/ListOperation.php:100]: You are right @basselhossam 

Thanks for the clarification 🙏 I will do the change accordingly, I agree that it gets more readable. 

Cheers

**Comment by jorgetwgroup** [comment]
OMG i didnt disable middleware so didnt see this problem:

screenshot-backpack test-2022 10 21-10_34_17

Everything work normal after get this branch

screenshot-backpack test-2022 10 21-10_37_33

I have disable middleware on base and Kernel

Screen Shot 2022-10-21 at 10 37 54

Look working great now.

Cheers.

**Pull request merged**